VolleyManagement.UnitTests/WebApi/ViewModels/Schedule/ScheduleViewModelTestFixture.cs

```diff
@@ -558,6 +560,167 @@ public ScheduleViewModelTestFixture WithThreeWeeksTwoDivisionsThreeGames()
             return this;
         }
 
+        public ScheduleViewModelTestFixture WithGamesInSeveralYearsAndWeeks()
+        {
+            _schedule.Schedule = new List<WeekViewModel>
+            {
+                new WeekViewModel
+                {
+                    Days = new List<ScheduleDayViewModel>
+                    {
+                        new ScheduleDayViewModel
+                        {
+                            Date = DateTime.Parse(DATE_A),
+                            Divisions = new List<DivisionTitleViewModel>
+                            {
+                                new DivisionTitleViewModel
+                                {
+                                    Id = 1,
+                                    Name = "Division Name",
+                                    Rounds = new List<int>
+                                    {
+                                        1,
+                                    },
+                                },
+                            },
+                            Games = new List<GameViewModel>
+                            {
+                                new GameViewModel
+                                {
+                                    Id = 1,
+                                    HomeTeamName = "TeamNameA",
+                                    AwayTeamName = "TeamNameB",
+                                    GameDate = DateTime.Parse(DATE_A).ToString("yyyy-MM-ddTHH:mm:sszzz"),
+                                    Result = new GameViewModel.GameResult
+                                    {
+                                        TotalScore = new UI.Areas.Mvc.ViewModels.GameResults.ScoreViewModel(3, 0),
```

Comment:
usings?

Reply:
It was a copy paste from another place. I believe we have ScoreViewModel in API area as well and to remove ambiguity